lib/licensed/sources/mix.rb

```diff
@@ -0,0 +1,396 @@
+# frozen_string_literal: true
+require "strscan"
+
+module Licensed
+  module Sources
+    class Mix < Source
+
+      LOCKFILE = "mix.lock"
+
+      # Returns whether a mix.lock is present
+      def enabled?
+        File.exists?(LOCKFILE)
```

Comment:
joining relative paths to `config.pwd` is preferable for clarity on the file location

```suggestion
        File.exists?(config.pwd.join(LOCKFILE))
```